### PR TITLE
News entries for exp(StridedMatrix) performance and five arg mul `UniformScaling`

### PR DESCRIPTION
**Pull request by jarlebring**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -192,6 +192,8 @@ Standard library changes
 * `det(M::AbstractMatrix{BigInt})` now calls `det_bareiss(M)`, which uses the
   [Bareiss](https://en.wikipedia.org/wiki/Bareiss_algorithm) algorithm to calculate precise
   values ([#40868]).
+* `exp(A::StridedMatrix)` has improved performance improvents  in terms of CPU-time and allocations ([#40668])
```
Comment (baggepinnen) [inline NEWS.md:195]:
```suggestion
* `exp(A::StridedMatrix)` has improved performance in terms of CPU-time and allocations ([#40668])
```

File: NEWS.md [hunk 2]
```diff
@@ -192,6 +192,8 @@ Standard library changes
 * `det(M::AbstractMatrix{BigInt})` now calls `det_bareiss(M)`, which uses the
   [Bareiss](https://en.wikipedia.org/wiki/Bareiss_algorithm) algorithm to calculate precise
   values ([#40868]).
+* `exp(A::StridedMatrix)` has improved performance in terms of CPU-time and allocations ([#40668])
+* Five argument `mul!` now supports in-place addition of `UniformScaling` ([#40731])
```
Comment (dkarrasch) [inline NEWS.md:196]:
```suggestion
* `exp(A::StridedMatrix)` has improved performance in terms of CPU-time and allocations ([#40668]).
* Five argument `mul!` now supports in-place addition of `UniformScaling` ([#40731]).
```

**Comment by dkarrasch** [comment]
@KristofferC Since this PR is against the release-1.7 branch, can I just merge it or does this require special care?

**Comment by ViralBShah** [comment]
Bump. What needs to be done here?

**Comment by jarlebring** [comment]
Too old to call news at this point.

**Comment by ViralBShah** [comment]
Usually, having the `news` label helps make sure that things make it into the news.

**Pull request closed without merging**